### PR TITLE
Fixing incorrect rendering of pipe character

### PR DESCRIPTION
**Pull request by SoniaLopezBravo**
This closes [#352](https://github.com/MicrosoftDocs/quantum-docs/issues/352)

### CODE REVIEW

**Comment by SoniaLopezBravo** [comment]
@tcNickolas I have changed it for another of the possible solutions. This one renders ok on GitHub files, hope it does on the docs page too.

**Comment by SoniaLopezBravo** [comment]
> Can you open a pull request with a matching change on the docs repo so that we get a preview to check whether this solution works indeed?

Hi @tcNickolas. I have created this [matching PR](https://github.com/MicrosoftDocs/quantum-docs-private/pull/569), in the preview we see that the pipe character is rendering ok :)

File: Specifications/Language/3_Expressions/PrecedenceAndAssociativity.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ Additional modifiers and combinators are listed further below and bind tighter t
 | --- | --- | --- | --- | --- |
 | [copy-and-update operator](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/CopyAndUpdateExpressions.md#copy-and-update-expressions) | `w/` `<-` | ternary | left  | 1  |
 | [range operator](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/ValueLiterals.md#range-literals) | `..` | infix | left | 2 |
-| [conditional operator](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/ConditionalExpressions.md#conditional-expressions) | `?` `\|` | ternary | right | 5 |
+| [conditional operator](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/ConditionalExpressions.md#conditional-expressions) | <code>?&vert;</code> | ternary | right | 5 |
```
Comment (tcNickolas) [inline Specifications/Language/3_Expressions/PrecedenceAndAssociativity.md:15]:
```suggestion
| [conditional operator](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/ConditionalExpressions.md#conditional-expressions) | <code>? &vert;</code> | ternary | right | 5 |
```

**Review by tcNickolas — DISMISSED** [review]
Great, looks like this solution works both on GitHub and in the docs!
Just one small comment: let's keep the whitespace between ? and |, since they are not a single operator like |||, but rather two parts of an operator that have an expression between them.

**Review by tcNickolas — APPROVED** [review]

**Comment by SoniaLopezBravo** [comment]
@bettinaheim could you review this?

**Review by alan-geller — APPROVED** [review]
Looks good to me.

**Comment by bettinaheim** [comment]
I believe this is addressed by the now merged PR https://github.com/microsoft/qsharp-language/pull/124. I'll hence close this.

**Pull request closed without merging**